src/api/protectedApiClient.ts

```diff
@@ -9,12 +9,12 @@ export interface ProtectedApiClient {
     currentPassword: string;
     newPassword: string;
   }) => Promise<void>;
-  readonly deleteUser: (request: { password: string }) => Promise<void>;
+  readonly decactivateAccount: () => Promise<void>;
 }
 
 export enum ProtectedApiClientRoutes {
   CHANGE_PASSWORD = '/api/v1/protected/user/change_password',
-  DELETE_USER = '/api/v1/protected/user/',
+  USER = '/api/v1/protected/user',
```

Comment:
This might be a bit overkill, but we could define another constant in our utils folder for our protected base route which can be `'api/v1/protected`

Reply:
Hmm, interesting. Could be nice. Not required.